crates/cxx-qt-gen/test_inputs/passthrough_and_naming.rs

```diff
@@ -37,7 +35,6 @@ pub mod ffi {
     extern "C" {}
 
     #[namespace = "namespace"]
-    #[custom_attr = "test"]
```

Comment:
this one should still work as it's not a CXX-Qt related block

Reply:
Are there any invalid attributes for there? that one originates in TypeNames::populate_from_foreign_mod_item, so I replaced like everywhere else, but that might need to just be attribute_get_path

Reply:
Ah right, so these are just CXX blocks so we should probably just pass through anything to CXX and let it deal with them. And in TypeNames yes use attribute_get_path to find any namespace attribute and just ignore the rest

Reply:
Cool, done